server/command_test.go

```diff
@@ -157,47 +157,47 @@ func TestPlugin_ExecuteCommand_Settings(t *testing.T) {
 		expectedMsg  string
 	}{
 		"no storage": {
-			commandArgs:  &model.CommandArgs{Command: "/jira instance settings", UserId: mockUserIDUnknown},
+			commandArgs:  &model.CommandArgs{Command: "/jira settings", UserId: mockUserIDUnknown},
```

Comment:
`/jira instance settings` is still a valid command right? Can we just add some tests for `/jira settings` instead of changing these tests?